### DISCUSSION TITLE
Which parameter in the response indicates the duration of the soccer competition?

### DISCUSSION THREAD

**Discussion by psnewer** [cloudbet/docs#71] [Q&A]
Which parameter in the response indicates the duration of the soccer competition?

**Comment by kgravenreuth** [comment]
Hi @psnewer ,

The [Competition endpoint](https://www.cloudbet.com/api/?urls.primaryName=Feed#/API-Version-2.0/GetCompetition) currently does not provide information about the duration of a given competition. 

However, we do provide the [Fixtures endpoint](https://www.cloudbet.com/api/?urls.primaryName=Feed#/API-Version-2.0/GetFixtures) which you can use to get a list of all fixtures for a given sport on a given day. If you filter this API endpoint's response further in your client code, then you could obtain the list of fixtures for a given competition on a given day, which might might meet your needs.

Let me know if this works or if you want to run any other ideas by us.

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
